config/manifests/bases/operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
```

Comment:
downstream I have this set to seamless upgrades... https://pkgs.devel.redhat.com/cgit/containers/rhdh-operator-bundle/tree/manifests/rhdhoperator.csv.yaml?h=rhdh-1.0-rhel-9#n8

Reply:
The operator doesn't support seamless upgrades at this moment. It will be fixed soon though. See: https://github.com/tumido/janus-operator#known-issues